### PR TITLE
Pull in d2l-switch-visibility from core.

### PR DESCRIPTION
**Pull request by dbatiste**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1105,7 +1105,7 @@
       }
     },
     "@brightspace-ui-labs/multi-select": {
-      "version": "github:BrightspaceUILabs/multi-select#a1d718a9a6240a3fb28bef64d1d205763b9147f4",
+      "version": "github:BrightspaceUILabs/multi-select#f424bfb0a657e5b44cdb371d1192d74e059a947b",
```
Comment (dbatiste) [inline package-lock.json:1108]: fyi: @jameswklassen

File: package-lock.json [hunk 2]
```diff
@@ -1164,9 +1164,9 @@
       }
     },
     "@brightspace-ui/intl": {
-      "version": "3.0.8",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/intl/-/intl-3.0.8.tgz",
-      "integrity": "sha512-mstURrd8MYKEOjOG3tDiemmHS9iOc4Yr33Ojh39qFYbeMeD1OKuDvQokLuhz10eukUt1pdfu1aqy5zrO98qErA==",
+      "version": "3.0.9",
```
Comment (dbatiste) [inline package-lock.json:1167]: fyi: @dlockhart

File: package-lock.json [hunk 3]
```diff
@@ -4406,17 +4368,16 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#829a3cea8a4971a35599b1ac590c115c35e66a68",
+      "version": "github:BrightspaceHypermediaComponents/activities#f21763c4b6da6e57d615eabcbff6d4f0dcd22da2",
```
Comment (dbatiste) [inline package-lock.json:4371]: fyi: @pureooze

File: package-lock.json (hunk 2 above)
Comment (dlockhart) [reply package-lock.json:1167]: Thanks -- that's fine it was a dummy release just to test out semantic-release.

File: package-lock.json (hunk 3 above)
Comment (dbatiste) [reply package-lock.json:4371]: @dlockhart (fastdom)

File: package-lock.json [hunk 4]
```diff
@@ -4758,7 +4719,7 @@
       }
     },
     "d2l-enrollments": {
-      "version": "github:BrightspaceHypermediaComponents/enrollments#73631afa3c7d03bed1683f22fdabe80fa03f3bbb",
+      "version": "github:BrightspaceHypermediaComponents/enrollments#6373a4e65f1d7a2d7091c8664c900a3c864260fc",
```
Comment (dbatiste) [inline package-lock.json:4722]: fyi: @dlockhart (fastdom)

File: package-lock.json [hunk 5]
```diff
@@ -4951,7 +4912,7 @@
       }
     },
     "d2l-localize-behavior": {
-      "version": "github:BrightspaceUI/localize-behavior#3257e7533197d29636eaf43be599f054df0fbd46",
+      "version": "github:BrightspaceUI/localize-behavior#ff9514e84470665348ccbb8584c32b5f51abf13b",
```
Comment (dbatiste) [inline package-lock.json:4915]: fyi: @dlockhart (intl message format stuff)

File: package-lock.json [hunk 6]
```diff
@@ -5087,7 +5048,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#95e358520fdc93f4691b2b8337546ecf9c8d80b4",
+      "version": "github:BrightspaceHypermediaComponents/organizations#b4cdb75e90ada5be0d47caa78f0d94c58a5be76b",
```
Comment (dbatiste) [inline package-lock.json:5051]: fyi: @dlockhart (fastdom)

haha... sorry if this is spamming you.

**Pull request merged**